alerts/large.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+
+const Large = (props) => {
+      const { 
+            userName,
+            userAddress,
+            userPhone,
+            keyword,
+            contactName,
+             
+
+      } 
+
+      return (
+            <div>
+                  Hi {contactName},
+                  <br/>
+                  You are receiving this alert from {userName} via the Pizza Now app because of a domestic violence situation that requires immediate action. They are currently located at {userAddress} and you can keep track of their location using *geolocation*. {userName} is in danger and requires immediate help. Please contact the authorities and make your way to {userName}'s location.
```

Comment:
I think instead of saying they're currently located at the address on their profile, we should directly put where're currently located according to geolocation

Reply:
Yeah, maybe both? Cause the geolocation won't show the apt number